screenshot.py

```
@@ -1,143 +1,704 @@
+            else:
+                debug(f'[gnome-screenshot] File not created')
+                return False
+
+        except subprocess.TimeoutExpired:
+            error('[gnome-screenshot] Screenshot timed out')
+            return False
+        except Exception as e:
+            error(f'[gnome-screenshot] Error: {e}')
+            return False
+
+    def capture_area_sync(self, filename: str, x: int, y: int,
```

Ruff (0.14.7)

Unused method argument: `x`

(ARG002)